### DISCUSSION TITLE
expose npm login custom message

### DISCUSSION THREAD

**Discussion by joebnb** [verdaccio/verdaccio#3687] [Ideas]
Current if npm login fail it will throw message in terminal

![image](https://user-images.githubusercontent.com/1646883/217702112-38583598-d051-44e0-970b-638309fbf547.png)

expect out put error message by `getUnauthorized` message.

![image](https://user-images.githubusercontent.com/1646883/217702370-b9cc8ab7-91c2-4d62-9e3f-622c895063d7.png)

**Comment by juanpicado** [comment]
It's a bug or a feature request? I'm not clear. Anyhow  what's the error specifically? Could you be more specific ?
